scripts/release-verifier-install-gate.sh

```
@@ -0,0 +1,456 @@
+	printf 'release verifier install gate: verifier source ref %s does not match inventory version %s\n' "$ts_ref" "$ts_version" >&2
+	exit 2
+fi
+if ! verifier_commit="$(git -C "$ROOT_DIR" rev-parse --verify "$ts_ref^{}" 2>/dev/null)"; then
+	printf 'release verifier install gate: required verifier release tag is missing: %s\n' "$ts_ref" >&2
+	exit 2
+fi
+if [[ ! "$ts_source_commit" =~ ^[0-9a-f]{40}$ || ! "$rust_source_commit" =~ ^[0-9a-f]{40}$ ]]; then
+	printf 'release verifier install gate: inventory must pin the 40-character peeled commit for %s before the product release can proceed\n' "$ts_ref" >&2
+	exit 2
+fi
+if [[ "$ts_source_commit" != "$rust_source_commit" || "$ts_source_commit" != "$verifier_commit" ]]; then
```

Shellcheck (0.11.0)

[warning] You probably wanted && here, otherwise it's always true.

(SC2055)